ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -73,6 +73,7 @@ descriptor:
   electric: "Process or component driven by electricity (as to distinguish from natural gas, for instance)."
   email: "A unique identifier address in the Internet."
   emergency : "a situation that poses an immediate risk to health, life property,  or environment."
+  entering: " The process of going into a thing (e.g. entering a heat exchanger)."
```

Comment:
for my own understanding, how is entering different from `input`?

Reply:
in current use entering and leaving are typically used in the case of measurements immediately on either side of a process stage and therefore generally refer to media that is physically moved and not consumed.  Input currently has no uses in actual fields, but you could imagine it to refer to something like a voltage on a terminal.